.changeset/thin-oranges-lick.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/react": patch
+---
+
+v38: Move pointerbox to deprecated still maintaining the main export
```

Comment:
is there a reason we want to maintain the main export or can we move to deprecated under v38 major?